src/Agent/NewRelic/Agent/Core/Aggregators/SpanEventAggregatorInfiniteTracing.cs

```diff
@@ -76,6 +76,8 @@ private void AgentConnected(AgentConnectedEvent _)
 
                 if(IsServiceEnabled)
                 {
+
+                    Log.Info($"SpanEventAggregatorInfiniteTracing: Configuration is invalid - Infinite Tracing will NOT be enabled.");
```

Comment:
We may want to change this log message to something else, because `IsServiceEnabled` could be false even if the configuration is valid (such as when DT is disabled).

Reply:
hmmmm, I am open here.  But felt that write log messages if they aren't using the feature.

Reply:
I just looked at the rest of the code in this method and you are right, that log message is appropriate.